### PR TITLE
Part 3: Update Maps Config

### PR DESCRIPTION
**Pull request by QSparks**
This PR includes updates to map components and configurations, part of the larger fnlf-cmip6-release. Updates include setting a maxZoom limit and updating the color and weight of polygon region boundaries. 
Demo link: https://services.pacificclimate.org/dev/plan2adapt/app/

### CODE REVIEW

File: src/components/maps/DataMap/DataMap.js [hunk 1]
```diff
@@ -84,28 +84,28 @@ class DataMapDisplay extends React.Component {
...
       xy: event.layerPoint,
     })
-    .then(response => {
-      const layerInfo = xml2js(response.data, {
-        compact: true,
+      .then(response => {
+        const layerInfo = xml2js(response.data, {
+          compact: true,
+        });
+        const value = layerInfo.foo;
```
Comment (corviday) [inline src/components/maps/DataMap/DataMap.js:91]: I know you didn't name this, just formatted, the code, but "foo" is not a terribly enlightening attribute name.

**Review by corviday — APPROVED** [review]
Looks pretty straightforward, a couple of attribute change and a bunch of code formatting. You demonstated the new region border colour yesterday, I think, so I'm reasonably confident it works.

**Review by corviday — APPROVED** [review]
Colour updates look straightforward. I think you mentioned that RCI had requested some changes based on accessibility to colourblind people. I'd be interested in knowing more about that, sometime, since I also select colour palettes for maps. :) Could you make a confluence page or something about the process sometime, share the wisdom?

**Comment by QSparks** [comment]
> Colour updates look straightforward and good to me. I think you mentioned that RCI had requested some changes based on accessibility to colourblind people. I'd be interested in knowing more about that, sometime, since I also select colour palettes for maps. :) Could you make a confluence page or something about the process sometime, share the wisdom?

A page for cartographic best practices is a great idea. In the meantime, have you heard of [color brewer](https://colorbrewer2.org/#type=diverging&scheme=BrBG&n=9)? It’s great for a quick reference of colour palettes and includes information on which ones are colour-blind friendly.

**Pull request merged**